app/models/bestuursorgaan.js

```diff
@@ -38,6 +38,28 @@ export default class BestuursorgaanModel extends Model {
   })
   bevat;
 
+  async classificatieUri() {
+    const bestuursorgaan = await this.isTijdsspecialisatieVan;
+    return bestuursorgaan
+      ? await bestuursorgaan.classificatie.uri
+      : await this.classificatie.uri;
+  }
+
+  // TODO decretale bestuursorganen should be fetched from database
```

Comment:
You also have a PR for the decretale concept schemes, so why isn't this fetched from the database?